pkg/prowgen/jobbase.go

```diff
@@ -131,13 +131,19 @@ func NewProwJobBaseBuilderForTest(configSpec *cioperatorapi.ReleaseBuildConfigur
 		if configSpec.Releases != nil {
 			p.PodSpec.Add(CIPullSecret())
 		}
+		if clusterType := test.MultiStageTestConfigurationLiteral.ClusterProfile.ClusterType(); clusterType != "" {
+			p.WithLabel(cioperatorapi.CloudLabel, clusterType)
```

Comment:
One more thing, while ideating in DPTP-2649, it looks like we could also use a label that carries the actual *cluster profile* value (which is a data proxy for the "what account will the job use"), could you add it?

Reply:
Will follow up tomorrow.